apis/python/src/tiledbsoma/__init__.py

```diff
@@ -236,6 +237,9 @@
     "get_implementation_version",
     "get_libtiledbsoma_core_version",
     "get_storage_engine",
+    "logging_level_debug",
```

Comment:
having these as public functions is really not how the Python logging system is supposed to be configured.  Typical best practices for a package is to document the logger name (typically it is the package or module name), and then let the application (not the package) configure logging with the standard `logging` package (e.g., a [logging config](https://docs.python.org/3/howto/logging.html#configuring-logging)).

In my view, we should deprecate (and eventually remove) these, as they will step on any application-specified logging config.

Reply:
More concretely - what I suggest is that we leave the names as they were, deprecate them, and remove in the next release.

Reply:
Do you want me to close this PR and instead create a ticket for deprecating the logging as it is now and revisit it in a holistic approach in the near future?

Reply:
> Do you want me to close this PR and instead create a ticket for deprecating the logging as it is now and revisit it in a holistic approach in the near future?

Yes, please do that and set it to triage. Add a link to the [logging config](https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library) documentation that @bkmartinjr linked above.